src/DeviceInterface.js

```diff
@@ -192,10 +204,9 @@ class AppleDeviceInterface extends InterfacePrototype {
 
         this.refreshAlias = () => window.webkit.messageHandlers['emailHandlerRefreshAlias'].postMessage({})
 
-        // TODO: deprecated
         this.isDeviceSignedIn = () => sendAndWaitForAnswer(() =>
             window.webkit.messageHandlers['emailHandlerCheckAppSignedInStatus'].postMessage({}),
-        'checkExtensionSignedInCallback'
+        'emailHandlerCheckAppSignedInStatusResponse'
```

Comment:
Changed this to conform to the new expectations that handler responses are in the format `something` → `somethingResponse`.